monitoring/shutdown-javaagent/src/test/java/com/google/cloud/runtimes/monitoring/shutdown/AgentTest.java

```diff
@@ -0,0 +1,32 @@
+package com.google.cloud.runtimes.monitoring.shutdown;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+
+public class AgentTest {
+
+
+  @Test
+  public void hasRegisteredShutdownHook() throws NoSuchFieldException, ClassNotFoundException,
```

Comment:
Could you add a comment for `// Arrange`, `// Act`, and `// Assert` blocks? I'm having trouble identifying what is the method under test and what is set up / clean up.